barretenberg/cpp/src/barretenberg/ultra_honk/relation_correctness.test.cpp

```diff
@@ -351,19 +348,18 @@ TEST_F(UltraRelationCorrectnessTests, Mega)
     ensure_non_zero(proving_key.polynomials.return_data_read_counts);
     ensure_non_zero(proving_key.polynomials.return_data_inverses);
 
-    // Construct the round for applying sumcheck relations and results for storing computed results
-    using Relations = typename Flavor::Relations;
     auto& prover_polynomials = instance->proving_key.polynomials;
     auto params = instance->relation_parameters;
 
     // Check that each relation is satisfied across each row of the prover polynomials
-    check_relation<Flavor, std::tuple_element_t<0, Relations>>(circuit_size, prover_polynomials, params);
-    check_relation<Flavor, std::tuple_element_t<1, Relations>>(circuit_size, prover_polynomials, params);
-    check_relation<Flavor, std::tuple_element_t<2, Relations>>(circuit_size, prover_polynomials, params);
-    check_relation<Flavor, std::tuple_element_t<3, Relations>>(circuit_size, prover_polynomials, params);
-    check_relation<Flavor, std::tuple_element_t<4, Relations>>(circuit_size, prover_polynomials, params);
-    check_relation<Flavor, std::tuple_element_t<5, Relations>>(circuit_size, prover_polynomials, params);
-    check_relation<Flavor, std::tuple_element_t<6, Relations>>(circuit_size, prover_polynomials, params);
-    check_linearly_dependent_relation<Flavor, std::tuple_element_t<7, Relations>>(
-        circuit_size, prover_polynomials, params);
+    check_relation<UltraArithmeticRelation<FF>>(circuit_size, prover_polynomials, params);
+    check_relation<UltraPermutationRelation<FF>>(circuit_size, prover_polynomials, params);
+    check_relation<DeltaRangeConstraintRelation<FF>>(circuit_size, prover_polynomials, params);
+    check_relation<EllipticRelation<FF>>(circuit_size, prover_polynomials, params);
+    check_relation<AuxiliaryRelation<FF>>(circuit_size, prover_polynomials, params);
+    check_relation<EccOpQueueRelation<FF>>(circuit_size, prover_polynomials, params);
+    check_relation<Poseidon2ExternalRelation<FF>>(circuit_size, prover_polynomials, params);
```

Comment:
Adding the new log deriv lookup relation but also the poseidon relations which weren't ever added to this suite.